eng/packaging.targets

```diff
@@ -20,6 +20,16 @@
     <UseIntellisenseDocumentationFile Condition="'$(UseIntellisenseDocumentationFile)' == '' and Exists('$(LibIntellisenseDocumentationFilePath)')">true</UseIntellisenseDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(ServicingVersion)' != ''">
+    <!-- Always update the package version in servicing. -->
+    <PackageVersion>$(MajorVersion).$(MinorVersion).$(ServicingVersion)</PackageVersion>
+    <IsNetCoreApp Condition="$(NetCoreAppLibrary.Contains('$(AssemblyName);'))">true</IsNetCoreApp>
+    <IsWindowsDesktop Condition="$(WindowsDesktopCoreAppLibrary.Contains('$(AssemblyName);'))">true</IsWindowsDesktop>
```

Comment:
Maybe we should move this to where we calculate the IsNetCoreApp property already. https://github.com/dotnet/runtime/blob/22cee85aa08061e5b2c1e893797a860e3c642e6c/src/libraries/Directory.Build.targets#L42

Reply:
Keeping it here avoids the unnecessary Contains functions running when they aren't needed (i.e. outside of servicing).